packages/protocol/contracts/common/Accounts.sol

```diff
@@ -0,0 +1,483 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+import "./interfaces/IAccounts.sol";
+
+import "../common/Initializable.sol";
+import "../common/Signatures.sol";
+import "../common/UsingRegistry.sol";
+
+contract Accounts is IAccounts, ReentrancyGuard, Initializable, UsingRegistry {
+
+  using SafeMath for uint256;
+
+  struct Signers {
+    // The address that is authorized to sign a vote on behalf of the account.
+    // The account can vote as well, whether or not an vote signing key has been specified.
+    address voting;
+    // The address that is authorized to sign consensus messages on behalf of the account.
+    // The account can manage the validator, whether or not an validation signing key has been
+    // specified. However if an validation signing key has been specified, only that key may
+    // actually participate in consensus.
+    address validating;
+
+    // The address of the key with which this account wants to sign attestations on the Attestations
+    // contract
+    address attesting;
+  }
+
+  struct Account {
+    bool exists;
+    // Each account may authorize signing keys to use for voting, valdiating or attestation.
+    // These keys may not be keys of other accounts, and may not be authorized by any other
+    // account for any purpose.
+    Signers signers;
+
+    // The address at which the account expects to receive transfers
+    address walletAddress;
+
+    string name;
+
+    // The ECDSA public key used to encrypt and decrypt data for this account
+    bytes dataEncryptionKey;
+
+    // The URL under which an account adds metadata and claims
+    string metadataURL;
+  }
+
+  mapping(address => Account) private accounts;
+
+  // Maps voting and validating keys to the account that provided the authorization.
+  mapping(address => address) public authorizedBy;
+
+
+  event AttestationSignerAuthorized(address indexed account, address signer);
+  event VoteSignerAuthorized(address indexed account, address signer);
+  event ValidationSignerAuthorized(address indexed account, address signer);
+
+  event AccountDataEncryptionKeySet(
+    address indexed account,
+    bytes dataEncryptionKey
+  );
+
+  event AccountNameSet(
+    address indexed account,
+    string name
+  );
+
+
+  event AccountMetadataURLSet(
+    address indexed account,
+    string metadataURL
+  );
+
+  event AccountWalletAddressSet(
+    address indexed account,
+    address walletAddress
+  );
+
+  event AccountCreated(
+    address indexed account,
+    string name
+  );
+
+  function initialize() external initializer {
+    _transferOwnership(msg.sender);
```

Comment:
What is ownable about this contract that needs to be governed?

Reply:
Blind copying on my side. Indeed nothing to govern here